fastapi/encoders.py

```diff
@@ -94,15 +126,21 @@ def jsonable_encoder(
         errors.append(e)
         try:
             data = dict(obj)
+            if not data:
+                return data
```

Comment:
As I understand it, if a user returns a DB model of some type, without using a Pydantic model with ORM mode, then this would convert to `dict` the first level, but there could be nested objects that are not JSON serializable, generating errors.

That's the reason to currently also pass `data` through `jsonable_encoder`, to make sure its keys and values are also "JSON-able".

Reply:
Yes, so return result quickly  if and only if the value is just empty dict.